routes/api.js

```diff
@@ -0,0 +1,27 @@
+const router = require('express').Router();
+const controllers = require('../controllers');
+
+/**
+ * GET request for movie list
+ * @return  array of movies
+ */
+router.get('/movies', controllers.movie.readAll);
+
+/**
+ * GET request for specific movie
+ * @param   id  movie id
+ * @return      movie details
+ */
+router.get('/movies/:id', controllers.movie.readOne);
+
+router.get('/movies/genre/:id', controllers.movie.readByGenre);
+
+/**
+ * GET request for genre list
+ * @return  array of genres
+ */
+router.get('/genres', controllers.genre.readAll);
+
```

Comment:
JDoc comment, pls. Your ones for the other functions are good, so just like that.

```suggestion
/**
 * GET request for genres
 * @param   id  genre id
 * @return      array of genres?
 */
```